### PR TITLE
[CCI] Revert changes to chart themes

### PR DESCRIPTION
**Pull request by andreymyssak**
### Description
[2086980](https://github.com/opensearch-project/oui/commit/20869800a42d1f658b3a2d4bde4d9a7a4df13296) and [e9a63af](https://github.com/opensearch-project/oui/commit/e9a63afd05e4bb532f30fd9359a71704e15a8ded) have been reverted to not break integrations with OpenSearch Dashboards.

### Issues Resolved
https://github.com/opensearch-project/oui/issues/810

### Check List
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [X] All tests pass
  - [X] `yarn lint`
  - [X] `yarn test-unit`
- [X] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/oui/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by BSFishy** [comment]
@AMoo-Miki do we still want this? If it's not merged, OSD builds will be broken on main. I suppose I'm not super opposed to merging it, but we still don't officially support Elastic Charts, so having this code in the repo feels kinda wrong

**Comment by AMoo-Miki** [comment]
Logically, these changes should be reverted in 1.x branches only since they are breaking changes in the 1.x releases. By that logic, we shouldn't revert these in `main`.

**Comment by BSFishy** [comment]
> Logically, these changes should be reverted in 1.x branches only since they are breaking changes in the 1.x releases. By that logic, we shouldn't revert these in `main`.

These were never backported to `1.x`, so we should be fine. Good to close?

**Comment by AMoo-Miki** [comment]
> These were never backported to `1.x`, so we should be fine. Good to close?

I want to hear @joshuarrrr on this as it will impact his work the most.

**Comment by joshuarrrr** [comment]
> I suppose I'm not super opposed to merging it, but we still don't officially support Elastic Charts, so having this code in the repo feels kinda wrong

In OpenSearch Dashboards today, we actually do still have a dependency on Elastic Charts - it's the charting library used by TSVB visualizations and a couple OpenSearch project dashboard plugins: https://github.com/search?q=repo%3Aopensearch-project%2FOpenSearch-Dashboards%20%40elastic%2Fcharts&type=code

As for these chart themes, they get loaded as part of OpenSearch Dashboards `_base.scss`:
https://github.com/opensearch-project/OpenSearch-Dashboards/blob/b337bea2d13cc6c8330ba6f0326c51b53baad7f3/src/core/public/styles/_base.scss#L14

So it's not really safe to remove these until the "Migrate Elastic Chart-rendered components to Vega-Lite" section of https://github.com/opensearch-project/OpenSearch-Dashboards/issues/2819 is completed

File: src-docs/src/views/markdown_editor/markdown_editor_with_plugins.js [hunk 1]
```diff
@@ -246,7 +248,7 @@ const ChartMarkdownRenderer = ({ palette, categories }) => {
   return (
     <Chart size={{ height: 320 }}>
       <Settings
-        theme={[customColors]}
+        theme={[customColors, OUI_CHARTS_THEME_LIGHT]}
```
Comment (joshuarrrr) [inline src-docs/src/views/markdown_editor/markdown_editor_with_plugins.js:251]: Note to self - open issue to look into migrating/removing elastic chart usage from OUI itself.

**Review by joshuarrrr — COMMENTED** [review]
This all looks good - note that some of the theme files were renamed (such as https://github.com/AMoo-Miki/oui/blob/c7f7c70c0f106e88c8e322962bb21bb601104320/src/themes/oui-next/oui_next_colors_dark.scss) in https://github.com/opensearch-project/oui/pull/835/ - we'll want to make sure each theme file currently existing in `main` has the

```
import '../../src/themes/charts/theme'; 
```
line after the elastic charts theme import

**Comment by andreymyssak** [comment]
Once https://github.com/opensearch-project/oui/pull/855 is merged, this PR will be ready for review

**Comment by andreymyssak** [comment]
@BSFishy, can you advise me in which section of CHANGELOG.md I should put this change?

**Comment by BSFishy** [comment]
> @BSFishy, can you advise me in which section of CHANGELOG.md I should put this change?

I think if the original ones had changelog entries, we'd have put them in `Breaking Changes` and then in this one, we'd just remove those entries because this is just meant to revert those. I'll add `skip-changelog` because it doesn't make sense to have a changelog for this one

**Review by joshuarrrr — APPROVED** [review]

**Review by SergeyMyssak — APPROVED** [review]

**Comment by joshuarrrr** [comment]
No need for backports, as, to the best of my understanding, the original changes were never backported.

**Pull request merged**